package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bower-npm-resolver",
-  "version": "0.9.0",
+  "version": "0.9.1",
```

Comment:
The version number will be updated automatically once released, can you revert this line?